src/components/studies/StudyLeftNav.tsx

```diff
@@ -162,10 +172,23 @@ const StudyLeftNav: FunctionComponent<StudyLeftNavProps> = ({
               >
                 <img
                   src={typeOfIcon(index, sectionLink.path)[index]}
-                  className={classes.navIcon}
+                  className={clsx(
```

Comment:
what is the code that actually prevents it from being able to navigate to?

Reply:
line 149 is where all the nav sections are being disabled.